x/ccv/consumer/keeper/validators.go

```diff
@@ -100,13 +102,74 @@ func (k Keeper) ValidatorByConsAddr(sdk.Context, sdk.ConsAddress) stakingtypes.V
 	return stakingtypes.Validator{}
 }
 
+func (k Keeper) SetLargestSoftOptOutValidatorPower(ctx sdk.Context, power uint64) {
```

Comment:
The naming is very confusing: `SetLargestSoftOptOutValidatorPower` sets a uint64 under the key `SoftOptOutThresholdPowerKey`, while `GetSoftOptOutThreshold` returns a string param (representing a fraction). There is also `UpdateLargestSoftOptOutValidatorPower`, but then the getter is called `GetSoftOptOutThresholdPower`.

Reply:
Updated here https://github.com/cosmos/interchain-security/commit/059c75086c6a1ed611ea4c855ad6b2f02d3494e3